static/usage.html

```diff
@@ -1247,7 +1247,7 @@ <h2><a href="#banking-apps">Banking apps</a></h2>
 
                 <p>The hardware attestation feature is part of the Android Open Source Project and
                 is fully supported by GrapheneOS. SafetyNet attestation chooses to use it to
-                enforce using Google certified operating systems. However, app developers can use
+                enforce using Google-certified operating systems. However, app developers can use
```

Comment:
In the past, the project has always written this as "Google certified", so I think it's best to keep it that way here too.

Reply:
No problem!